schematic/store/synapse.py

```diff
@@ -65,18 +69,31 @@ class ManifestDownload(object):
     syn: synapseclient.Synapse
     manifest_id: str
 
-    def _download_manifest_to_folder(self):
+    def _download_manifest_to_folder(self) -> File:
         """
         try downloading a manifest to local cache or a given folder
         manifest
         Return: 
-            manifest_data: A new Synapse Entity object of the appropriate type
+            manifest_data: synapse file entity
```

Comment:
The type should be in parenthesis:

manifest_data(File): synapse file entity

Perhaps the description could be more descriptive as well like "A Synapse file entity of the downloaded manifest"

Reply:
Thanks Andrew. Let me update the comment like you suggested.